rust/sdk-examples/src/config/indexer_processor_config.rs

```diff
@@ -17,6 +17,7 @@ pub struct IndexerProcessorConfig {
     pub processor_config: ProcessorConfig,
     pub transaction_stream_config: TransactionStreamConfig,
     pub db_config: DbConfig,
+    pub backfill_config: Option<BackfillConfig>,
```

Comment:
Do we need to set defaults here so that the config still works even if `backfill_config` is not specified?

Reply:
Defaults not needed here. The parsing works without a specified `backfill_config` already.